README.md

```diff
@@ -75,32 +90,35 @@ Here is an example of how a re-occurring fragment is rendered.
 public final class String
     implements java.io.Serializable, Comparable<String>, CharSequence {
 
-    // #docfragment "standard-object-methods"
+    // #docfragment "Standard Object methods"
     public int hashCode() {
         // ...
         return hash;
     }
-    // #enddocfragment "standard-object-methods"
+    // #enddocfragment "Standard Object methods"
     
     /* here goes irrelevant code */
 
-    // #docfragment "standard-object-methods"
+    // #docfragment "Standard Object methods"
     public boolean equals(Object anObject) {
         // ...
         return false;
     }
-    // #enddocfragment "standard-object-methods"
+    // #enddocfragment "Standard Object methods"
 
     /* here goes more irrelevant code */
 
-    // #docfragment "standard-object-methods"
+    // #docfragment "Standard Object methods"
     public String toString() {
         return this;
     }
-    // #enddocfragment "standard-object-methods"
+    // #enddocfragment "Standard Object methods"
 }
 ```
 
+Note the usage of spaces in the fragment names. Thanks to putting fragment names into quotes, normal
```

Comment:
As the name of each fragment is put into quotes, space symbols may be used. So one may use the natural language (e.g. "My favorite fragment here!") instead of `CamelCase`, `snake_case`, or `kebab-case` (e.g. "my-favourite-fragment-here-exclamation-mark").